### DISCUSSION TITLE
Guidance on creating collections based on folder location with this script

### DISCUSSION THREAD

**Discussion by svtcobra1996** [mza921/Plex-Auto-Collections#243] [Q&A]
Hello, I'm pretty new to phython and scriptting.  I was looking at this to auto create Plex collections.  I am wondering if there is away to modify it so instead of using tmdb.org or imdb.com, etc that it can be done based on the file path location?
For example I have a folder called **Horror - Indie** which the movies may be in something like IMDB, but may not.  Would it be possible to do a dir or ls of the folder and write that to a text file, then based on that text file create a collection using this script?
If so could you offer some guidance on how to do so?

**Comment by meisnate12** [comment]
development here has stalled check out the updated [Plex Meta Manager](https://github.com/meisnate12/Plex-Meta-Manager).

**Reply by svtcobra1996** [reply]
ok thanks, i will post in there and see.  i was looking at that as well.
